tempo/serve/base.py

```diff
@@ -135,6 +138,7 @@ def __getstate__(self) -> dict:
         state["context"] = SimpleNamespace()
         # Remove the insights manager from the cloudpickle context
         state["insights_manager"] = SimpleNamespace()
+        state["state"] = SimpleNamespace()
```

Comment:
We can have these as something else other than `SimpleNamespace()` BTW. I think that the main reason I chose to use `SimpleNamespace` for `context` is that it's always gonna be a `SimpleNamespace`, so that's actually its default "empty" value. However, we could think of different values for the other keys.

Reply:
Good shout - i did think this made it more explicit, as None could end up being just not set (potenttially different problem)